mobile/lib/modules/settings/ui/advanced_settings/advanced_settings.dart

```diff
@@ -49,6 +54,15 @@ class AdvancedSettings extends HookConsumerWidget {
         ),
       ).tr(),
       children: [
+        SettingsSwitchListTile(
+          enabled: true,
+          appSettingService: appSettingService,
+          valueNotifier: offlineBrowsing,
+          settingsEnum: AppSettingsEnum.offlineBrowsing,
+          title: "Offline Browsing",
```

Comment:
Need advice on how this should be handled to support the translation service.. Should I just add the fields to the English json?